### DISCUSSION TITLE
Feature suggestion: Show total accrued time in each status for project cards

### DISCUSSION THREAD

**Discussion by glennsc** [community/community#62008] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

So many metrics questions (cycle time, throughput, etc) can be derived if you know how much time a card in a project has accrued in each status: 

![IMG_0140](https://github.com/community/community/assets/829567/f9de7e57-c5ac-4279-ab1a-dbdad7f5eacb)

I have seen the cool work on [issue metrics](https://github.blog/2023-07-19-metrics-for-issues-pull-requests-and-discussions/) however in projects those issue metrics ultimately roll up to a card. I care about the card as it moves through the status changes. 

The workaround I see is to webhook the card update events and accrue time in a separate data store but it’s a bit brittle; would love to see this in the product. 

Thanks for considering. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by glennsc** [comment, marked as answer]
This appears to be non-trivial to implement in user land until this `project_v2_field` web hook payload comes around, which per @skw is not prioritized.

https://github.com/orgs/community/discussions/17405#discussioncomment-6364482

**Comment by mariebakke** [comment]
I would LOVE this feature. It is so critical to being able to identify where we are having bottlenecks in our workflow.

**Comment by glennsc** [comment]
Well it appears this ability was implemented, via the `project_v2_field` hook:

https://github.blog/changelog/2024-06-27-github-issues-projects-graphql-and-webhook-support-for-project-status-updates-and-more/#using-webhooks-for-project-custom-field-changes
